mongo/description/shared_spec_test.go

```diff
@@ -93,6 +103,13 @@ func serverKindFromString(s string) ServerKind {
 		return RSGhost
 	case "Mongos":
 		return Mongos
+	case "LoadBalancer":
+		return LoadBalancer
+	case "PossiblePrimary", "Unknown":
+		// Go does not have a PossiblePrimary server type and per the SDAM spec, this type is synonymous with Unknown.
```

Comment:
See [ServerType](https://github.com/mongodb/specifications/blob/ec326862be443d11f0233cee2de32a95751144df/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#servertype).